docs/README-he.md

```diff
@@ -0,0 +1,187 @@
+| Russian              | [ramstore07](https://github.com/ramstore07), [dmtrshat](https://github.com/dmtrshat)                 |
+| Brazilian Portuguese | [JoaoPedroMoraes](https://github.com/JoaoPedroMoraes)                                                |
+| Turkish              | [3DShark](https://github.com/3DShark)                                                                |
+| Italian              | [gello94](https://github.com/gello94)                                                                |
+| Slovak               | [MartinT](https://github.com/MartinTuroci)                                                           |
+| Czech                | [PetrTodorov](https://github.com/PetrTodorov)                                                        |
+| Tagalog              | [giftofgrub](https://github.com/giftofgrub)                                                          |
+| Traditional Chinese  | [oxygen-TW](https://github.com/oxygen-TW)                                                            |
+| Swedish              | [PalleKarlsson](https://github.com/PalleKarlsson)                                                    |
+| Greek                | [Shuin-San](https://github.com/Shuin-San)                                                            |
+| Vietnamese           | [HaiDang666](https://github.com/HaiDang666)                                                          |
+| Finnish              | [cjola002-xamk](https://github.com/cjola002-xamk)                                                    |
+
+## תהליך פיתוח
+
+ראשית כל, וודאו שקראתם את [עמוד הסבר על התרומות](https://github.com/nukeop/nuclear/wiki/Contributing).
+
+השתמשו ב - npm:
+<div dir="ltr">
+
+```shell
+$ npm install # installs dependencies
+$ npm start
+```
+</div>
+
+חלון חדש אמור להיפתח אשר יטען את את התוכנה.
+
+---
+
+כדי לבנות למערכת הפעלה הנוכחית שלך:
+<div dir="ltr">
+
+```shell
+$ lerna bootstrap
+$ npm run build
+```
+</div>
+
+על מנת לבנות לכל מערכות ההפעלה החליפו את `build` עם `build:all`. הקבצים יהיו ב `packages/app/realease`
+
+---
+
+יש אפשרות גם להריץ סביבת פיתוח בעזרת Docker Containers, אבל זו אפשרות נסיונית.
+אתם צריכים Docker ו - Docker - Compose. אתם תצטרכו לאפשר למשתמש הראשי להתחבר ל - X11 Display, ואז תוכלו להריץ Docker - Compose:
+<div dir="ltr">
+
+```shell
+$ xhost SI:localuser:root
+$ sudo docker-compose up dev
+```
+</div>
+
+נכון לרגע זה, יש אפשרות גם לבנות גרסת flatpak. עליכם להתקין gobject-introspection, ו -  flatpak-builder. אחרי זה עליכם להתקין את הקבצים הדרושים בשביל תהליך הקימפול. אתם נדרשים לגרסת 19.08 ל - flatpaks האלה.
+<div dir="ltr">
+
+```shell
+$ flatpak install flathub org.freedesktop.Platform
+$ flatpak install flathub org.freedesktop.Sdk
+$ flatpak install flathub io.atom.electron.BaseApp
+```
+</div>
+
+כעת, על מנת לבנות את הפרוייקט (כתבו בנוסף את הפקודה `--verbose` כדי לקבל פלט מפורט יותר ):
```

Comment:
Suggest to fix the flag direction:
`כעת, על מנת לבנות את הפרוייקט (כתבו בנוסף את הפקודה <span dir="ltr">`--verbose`</span> כדי לקבל פלט מפורט יותר ):`